<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteDebugWindowClass</class>
 <widget class="QMainWindow" name="RemoteDebugWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RemoteDebugWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>1</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="categories">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabStateTree">
       <attribute name="title">
        <string>State Trees</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QTreeWidget" name="debugWidgets">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>464</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>16777215</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string notr="true">1</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="stateTree">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>250</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabScriptConsole">
       <attribute name="title">
        <string>Script Console</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="QTextBrowser" name="scriptOutput">
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="scriptInput">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>40</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabMap">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>1</number>
        </property>
        <item>
         <widget class="MapWidget" name="graphics2d">
          <property name="backgroundBrush">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>200</red>
             <green>200</green>
             <blue>200</blue>
            </color>
           </brush>
          </property>
          <property name="dragMode">
           <enum>QGraphicsView::ScrollHandDrag</enum>
          </property>
          <property name="transformationAnchor">
           <enum>QGraphicsView::AnchorUnderMouse</enum>
          </property>
          <property name="rubberBandSelectionMode">
           <enum>Qt::IntersectsItemBoundingRect</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="qmlView">
       <attribute name="title">
        <string>View</string>
       </attribute>
       <widget class="QDeclarativeView" name="declarativeView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>791</width>
          <height>521</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionConnect_LocalHost"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Close The Application</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionConnect_LocalHost">
   <property name="text">
    <string>Connect Localhost</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QDeclarativeView</class>
   <extends>QGraphicsView</extends>
   <header>QtDeclarative/QDeclarativeView</header>
  </customwidget>
  <customwidget>
   <class>MapWidget</class>
   <extends>QGraphicsView</extends>
   <header>mapwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="remotedebugwindow.qrc"/>
 </resources>
 <connections/>
</ui>
